core/__init__.py

```diff
@@ -0,0 +1 @@
+"""Core files for backend functionalities."""
```

Comment:
This doesn't need a docstring, it's fine (and probably better) for `__init__` files to be blank.

Reply:
This is from pydocstyle(PEP8)
https://media.readthedocs.org/pdf/pydocstyle/2.1.1/pydocstyle.pdf

D107 Missing docstring in __init__

Am I making the right call in trying to comply with both Pylint and PEP8?

Reply:
No, you don't need to follow all the pylint rules. We can make exclusions as needed. I think this is one of them, and the APP thing is another.

Reply:
Gotcha! Done :)